src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md

```diff
@@ -53,6 +46,118 @@ On Windows (Powershell)
+Install a Java version 11 JDK or later. We do not currently support Java 17 or later. The following example uses Homebrew to install Java version 11 on a MacOS.
+
+1. Install Java version 11 JDK using HomeBrew by running the following command in your terminal:
+```
+brew install openjdk@11
+```
+2. Set the JAVA_HOME environment variable to reference the JVM in which you want the test to run. To avoid setting this on every terminal, add the following information to your shell’s initialization file:
+```
+export JAVA_HOME=/path/to/your/jdk/Contents/Home
+```
+where `/path/to/your/jdk` is you JDK path. 
+You can find your JDK path by running the `which java` command in your terminal. 
+
+
+
+## Installing Maven
+
+Install Maven 3.8 or later. The following example uses Homebrew:
+
+1. Run the following command to install Maven:
+```
+brew update
+brew install maven
+```
+2. Add the follwing information to your `~/.m2/settings.xml` file:
+```
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+    <pluginGroups>
+        <pluginGroup>dev.galasa</pluginGroup>
+    </pluginGroups>
+     
+    <profiles>
+        <profile>
+            <id>galasa</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <repositories>
+                <repository>
+                    <id>maven.central</id>
+                    <url>https://repo.maven.apache.org/maven2/</url>
+                </repository>
+                <!-- To use the bleeding edge version of galasa, use the development obr
+                <repository>
+                    <id>galasa.repo</id>
+                    <url>https://development.galasa.dev/main/maven-repo/obr</url> 
+                </repository>
+                -->
+            </repositories>
+            <pluginRepositories>
+                <pluginRepository>
+                    <id>maven.central</id>
+                    <url>https://repo.maven.apache.org/maven2/</url>
+                </pluginRepository>
+                <!-- To use the bleeding edge version of galasa, use the development obr
+                <pluginRepository>
+                    <id>galasa.repo</id>    
+                    <url>https://development.galasa.dev/main/maven-repo/obr</url> 
+                </pluginRepository>
+                -->
+             </pluginRepositories>
+         </profile>
+     </profiles>
+</settings>
+```
+This information tells Maven where to find some custom Maven plug-in tools that are used by the build.
+
+
+## Installing Gradle
+
+1. Install <a href="https://gradle.org/install/" target="_blank"> Gradle</a> version 6.x. *Note:* Gradle version 7.x is not currently supported.
+2. Put Gradle on your PATH by adding the following information to your shell’s initialization file:
```

Comment:
I think this is fine. We are not telling people how to install gradle, just that it needs to be on the path.

Reply:
ok